src/header.js

```diff
@@ -0,0 +1,45 @@
+import homePage from './home';
+import contactPage from './contact';
+import menuPage from './menu';
+
+export const renderSection = (section) => {
+  const divSection = document.getElementById('pageContent');
+  divSection.innerHTML = '';
+  if (section === 'home') {
+    divSection.appendChild(homePage());
+  } else if (section === 'contact') {
+    divSection.appendChild(contactPage());
+  } else if (section === 'menu') {
+    divSection.appendChild(menuPage());
+  }
+};
+
+export function headers() {
```

Comment:
- [ ] It would be great if you use ES6 syntax here. For example, don't export a function before defining a function and use arrow functions.